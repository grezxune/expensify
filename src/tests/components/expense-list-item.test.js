import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/expense-list-item';
import expenses from '../fixtures/expenses';

test('should render ExpenseListItem with expense object', () => {
    const expense = expenses[1];
    const wrapper = shallow(<ExpenseListItem {...expense} key={expense.description} />);
    expect(wrapper).toMatchSnapshot();
});
