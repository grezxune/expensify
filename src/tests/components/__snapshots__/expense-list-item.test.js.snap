// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expense object 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    rent
  </Link>
  <p>
    1095
     - 
    -345600000
  </p>
</div>
`;
